packages/near-api-js/src/account.ts

```diff
@@ -629,4 +630,35 @@ export class Account {
             available: availableBalance.toString()
         };
     }
+
+    /**
+     * Returns total delgated stake of the account
+     * @returns {Promise<string>}
+     */
+    async getActiveDelegatedStakeBalance(): Promise<string>  {
+        const { current_validators, next_validators, current_proposals } = await this.connection.provider.validators(null);
+        const pools:Set<string> = new Set();
+        [...current_validators, ...next_validators, ...current_proposals]
+            .forEach((validator) => pools.add(validator.account_id));
+
+        const prefix = getValidatorRegExp(this.connection.networkId);
+        const promises = [...pools]
+            .filter((v) => v.indexOf('nfvalidator') === -1 && v.match(prefix))
+            .map(async (validator) => (
+                await this.viewFunction({
+                    contractId: validator,
+                    methodName: 'get_account_total_balance',
+                    args: { account_id: this.accountId },
+                })
```

Comment:
To avoid races, it is better to make the view-function calls on the same block. I would fetch the final block and use its hash in the view-function call (NEAR JSON RPC allows to specify the `block_id`: https://docs.near.org/api/rpc/contracts#call-a-contract-function)